### PR TITLE
transparent background and fix margins

### PR DESCRIPTION
**Pull request by tritus**

### CODE REVIEW

**Review by dany-kun — APPROVED** [review]

File: PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java [hunk 1]
```diff
@@ -598,13 +600,13 @@ private void createPages() {
         if (mPageMode == AUTO_PAGE_MODE &&
             mViewRect.surfaceW > mViewRect.surfaceH) {
             mPages[FIRST_PAGE] = new Page(mViewRect.left + marginLeft, 0,
-                                          mViewRect.top + marginTop, mViewRect.bottom - marginBottom);
+                                          mViewRect.top - marginTop, mViewRect.bottom + marginBottom);
             mPages[SECOND_PAGE] = new Page(0, mViewRect.right - marginRight,
```
Comment (JulienRepele) [inline PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java]: Why the first and the second page does not have the same size ?

Comment (JulienRepele) [reply PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java]: Ah it's because it's not size but position, right?

Comment (tritus) [reply PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java]: exactly

**Review by Mathieu-Heuzey — APPROVED** [review]

File: PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java (hunk 1 above)
Comment (JulienRepele) [reply PageFlip/src/main/java/com/eschao/android/widget/pageflip/PageFlip.java]: ok so maybe you could create some variable like `firstPageTopLeft` `firstPageBottomLeft`... to be more understandable.

**Review by JulienRepele — APPROVED** [review]

**Pull request merged**